src/scrapers/main.py

```diff
@@ -31,22 +36,34 @@ def docs():
         print('{: <16}{}'.format(s.code, s.name))
 
 
-(opts, args) = parser.parse_args()
+arguments = parser.parse_args()
 
-if opts.help:
+if arguments.help:
     docs()
 else:
-    print(f'Running {len(args)} scrapers. Use --help to see all individual scrapers')
+    print(f'Running {len(arguments.scrapers)} scrapers. Use --help to see all individual scrapers')
+    if arguments.group:
+        if arguments.group != InitiativeGroup.SUPPLY \
+                and arguments.group != InitiativeGroup.DEMAND:
+            print(f"Invalid group given {arguments.group}")
+            exit()
+
+    run_scrapers = None
+    if arguments.scrapers:
+        run_scrapers = reduce(lambda a, b: a + b, arguments.scrapers)
```

Comment:
Waarom zou je de lijst aan elkaar plakken? Je doet toch geen speciale string operaties verder

Reply:
Je kan meerdere keren -s doen. Dus `-s nlve -s wams mbmb` Dan bevat argument.scrapers een lijst van lijsten. er is een andere action: extend, maar die is pas in python 3.8 toegevoegd. Dus leek me beter het even zo te doen.